hbase-client/src/main/java/org/apache/hadoop/hbase/client/RawAsyncHBaseAdmin.java

```diff
@@ -3279,13 +3279,22 @@ private void getProcedureResult(long procId, CompletableFuture<Void> future, int
 
   @Override
   public CompletableFuture<Boolean> normalize() {
+    return normalize(RequestConverter.buildNormalizeRequest());
+  }
+
+  @Override
+  public CompletableFuture<Boolean> normalize(NormalizeTableFilterParams ntfp) {
+    return normalize(RequestConverter.buildNormalizeRequest(ntfp));
+  }
+
+  private CompletableFuture<Boolean> normalize(NormalizeRequest request) {
     return this
-        .<Boolean> newMasterCaller()
-        .action(
-          (controller, stub) -> this.<NormalizeRequest, NormalizeResponse, Boolean> call(
-            controller, stub, RequestConverter.buildNormalizeRequest(),
-            (s, c, req, done) -> s.normalize(c, req, done), (resp) -> resp.getNormalizerRan()))
-        .call();
+      .<Boolean> newMasterCaller()
+      .action(
+        (controller, stub) -> this.<NormalizeRequest, NormalizeResponse, Boolean> call(
```

Comment:
nit: we can remove type arg <>

Reply:
The params on `call` can be elided but the param on `newMasterCaller()` is mandatory.